hacking/aws_config/testing_policies/security-policy.json

```diff
@@ -91,22 +91,6 @@
                 "arn:aws:logs:{{aws_region}}:{{aws_account}}:log-group:ansible-testing*"
             ]
         },
-        {
-            "Sid": "AllowSTSAnsibleTests",
-            "Action": [
-                "iam:CreateRole",
-                "iam:DeleteRole",
-                "iam:DetachRolePolicy",
-                "sts:AssumeRole",
```

Comment:
The other permissions being removed here seem to be present in the rest of the file, except this one. It seems to be at least used by the sts_assume_role tests.